package.json

```diff
@@ -17,7 +17,12 @@
     "lint:watch": "npm run lint -- --watch",
     "prepublish": "npm run lint && npm run test && npm run build",
     "test": "mocha --compilers js:babel-register src/**/*-test.js",
-    "test:watch": "npm run test -- --watch src/**/*-test.js"
+    "test:watch": "npm run test -- --watch src/**/*-test.js",
+    "docs:clean": "rimraf _book",
+    "docs:prepare": "gitbook install",
+    "docs:build": "npm run docs:prepare && gitbook build -g acdlite/redux-actions",
+    "docs:watch": "npm run docs:prepare && gitbook serve",
+    "docs:publish": "npm run docs:clean && npm run docs:build && cp CNAME _book && cd _book && git init && git commit --allow-empty -m 'update book' && git checkout -b gh-pages && touch .nojekyll && git add . && git commit -am 'update book' && git push git@github.com:acdlite/redux-actions gh-pages --force"
```

Comment:
OK it's up to you, we can leave it out for now. By the way, how will this work if I want to publish new documentation, since it's hosted in your gitbook URL? Do we need to grant access to developers of this repo until @acdlite sets up pushing to GitHub pages (I don't have access to the repository settings)?